verkle_trie_eip/verkle_trie.py

```diff
@@ -1050,128 +1050,144 @@ def compute_update(path, node):
 
     values = {}
 
-    for i in range(NUMBER_STEMS):
-        stem = randint(0, 2**248-1).to_bytes(31, "little")
-        for i in range(CHUNKS_PER_STEM):
-            key = stem + bytes([randint(0, 2**8-1)])
-            value = randint(0, 2**256-1).to_bytes(32, "little")
-            update_verkle_tree_nocommitmentupdate(root_node, key, value)
-            values[key] = value
+    # Build a tree with a single key (0..0).
+    key = bytes([0] * 32)
+    value = randint(0, 2**256-1).to_bytes(32, "little")
+    update_verkle_tree_nocommitmentupdate(root_node, key, value)
+    values[key] = value
+    verkle_add_missing_commitments(root_node)
+
+    # Create two keys that will end up in the leaf node, but both have different stems.
+    proof_key1 = bytes([0, 1] + [0] * 32)
+    proof_key2 = bytes([0, 2] + [0] * 32)
+    depths, extension_present, commitments_sorted_by_path_serialized, other_stems, D_serialized, ipa_proof = make_verkle_proof(root_node, [proof_key1, proof_key2])
+    print("OtherStems has length {0}".format(len(other_stems)))
+    print("Extensions has length {0}".format(len(extension_present)))
```

Comment:
Mostly commented the existing main, and wrote this small test-case.

The output of this run is:
```
   Starting proof computation
   Computed key paths           0.000 s
   Sorted all commitments       0.000 s
   Hashed to r                  0.000 s
   Computed g polynomial        0.001 s
   Computed commitment D        0.005 s
   Computed h polynomial        0.000 s
   Computed IPA proof           0.060 s
   Serialized commitments       0.000 s
OtherStems has length 1
Extensions has length 2
```

Note `Extensions has length 2`